py/rbv.py

```
@@ -0,0 +1,56 @@
+    if r.returncode != 0:
+        sys.exit(r.returncode)
+
+    # Send 'E' keystroke after brief delay to switch to tree view
+    pane_id_result = subprocess.run(
+        ["tmux", "display-message", "-p", "#{pane_id}"],
+        capture_output=True,
+        text=True,
+    )
+    if pane_id_result.returncode == 0:
+        pane_id = pane_id_result.stdout.strip()
+        subprocess.Popen(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)